psi4/src/psi4/libfock/PKmanagers.cc

```diff
@@ -437,7 +437,7 @@ void PKManager::make_J_vec(std::vector<SharedMatrix> J) {
         // Shared pointer initialized to null
         //TODO Could actually store triangle since J is always symmetric
         } else {
-            JK_vec_.push_back(nullptr);
+            JK_vec_.push_back(NULL);
         }
```

Comment:
Yes, you're right, strictly speaking it could be better to keep ``nullptr``. Feel free to do it since I have not updated my forks yet...